lib/iris/tests/integration/test_grib2.py

```diff
@@ -280,6 +282,7 @@ def test_regular(self):
         cube = load_cube(path)
         self.assertCMLApproxData(cube)
 
+    @skip('iris-grib is currently causing failures')
     def test_reduced(self):
```

Comment:
This failure seems to have been introduced by the latest verion of iris-grib. The points of one of the dimensions is reading [0.] rather than [~2.] .

Reply:
When loading this file I got the warning:
```
UserWarning: Unable to create instance of HybridPressureFactory.
The source data contains no field(s) for 'ref_surface_pressure'.
```
Possibly related to this commit: https://github.com/SciTools/iris-grib/commit/9f26e7aa23d15e93356598b4c231db887abdcd46

Reply:
This is actually related to this issue: https://github.com/SciTools/iris-grib/issues/150
When the suggested changes are applied this test passes again.